sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/KafkaCosmosConfig.java

```diff
@@ -50,16 +51,96 @@ public class KafkaCosmosConfig extends AbstractConfig {
     private static final String APPLICATION_NAME_DOC = "Application name. Will be added as the userAgent suffix.";
     private static final String APPLICATION_NAME_DISPLAY = "Application name.";
 
+    // Throughput control config
+    private static final String THROUGHPUT_CONTROL_ENABLED = CONFIG_PREFIX + "throughputControl.enabled";
```

Comment:
The config here will be applied to all containers. 
since in the source/sink, we allow define multiple containers, so I am wondering should we also allow the customer to define different config for different container.

Reply:
What is the downside of having this config enabled for all containers? 
Ideally we should, however, we can do that post GA, thoughts?